params/protocol_params.go

```diff
@@ -127,8 +127,9 @@ const (
 	ElasticityMultiplier     = 2          // Bounds the maximum gas limit an EIP-1559 block may have.
 	InitialBaseFee           = 1000000000 // Initial base fee for EIP-1559 blocks.
 
-	MaxCodeSize     = 24576           // Maximum bytecode to permit for a contract
-	MaxInitCodeSize = 2 * MaxCodeSize // Maximum initcode to permit in a creation transaction and create instructions
+	MaxCodeSize         = 24576 // Maximum bytecode to permit for a contract
+	MaxCodeSizeShanghai = 32768
```

Comment:
Should we apply comment here `// Maximum bytecode to permit for a contract after Shanghai Hardfork`

Reply:
added